src/components/molecules/AdminManagementForm.tsx

```diff
@@ -0,0 +1,199 @@
+import { observer } from 'mobx-react-lite';
+import React from 'react';
+import DialogWithHeader from './DialogWithHeader';
+import { useTranslation } from 'react-i18next';
+import { makeStyles, withStyles, createStyles } from '@material-ui/core/styles';
+import { Typography } from 'components/atoms';
+import {
+  TableContainer,
+  Table,
+  Box,
+  InputLabel,
+  MenuItem,
+  Select,
+  FormControl,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Theme,
+  ButtonGroup,
+  Button,
+} from '@material-ui/core';
+import { silverGrayColor, blackColor } from 'style';
+import {
+  cancelEditModeHelper,
+  saveEditModeHelper,
+  changeEditObjectHelper,
+  changeCurrentSelectedRoleHelper,
+  IeditObjList,
+  IProps,
+} from './AdminManagementFormHelpers';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  table: {
+    border: `1px solid ${blackColor}`,
+    borderBottom: 0,
+    borderCollapse: 'separate',
+  },
+  saveButton: {
+    marginTop: '20px',
+    marginBottom: '10px',
+  },
+}));
+const StyledTableCell = withStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      textAlign: 'center',
+      padding: theme.spacing(0.7, 0),
+      borderBottom: `1px solid ${blackColor}`,
+    },
+    sizeSmall: {
+      '&:last-child': {
+        padding: theme.spacing(0.7, 1),
+      },
+    },
+    head: {
+      backgroundColor: silverGrayColor,
+      color: theme.palette.common.black,
+    },
+  }),
+)(TableCell);
+const StyledTableRow = withStyles(() =>
+  createStyles({
+    root: {
+      // styles can be added here
+    },
+  }),
+)(TableRow);
+
+const AdminManagementForm: React.FC<IProps> = ({ labels, defaultValues, isShowing, onClose }) => {
+  const [editObject, setEditObject] = React.useState<IeditObjList>({});
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  const EditMode: any = (props: any) => {
+    return (
+      <ButtonGroup>
+        <Button variant="contained" onClick={() => saveEditMode(props.itemData)}>
+          שמור
```

Comment:
use i18 support , example :`title={t('usersManagement.title')}` instead שמור and בטל

Reply:
@danielsh28 all right, i fixed that